apps/eec/fields/index.js

```diff
@@ -45,6 +45,82 @@ module.exports = {
       className: 'govuk-!-margin-bottom-6'
     }
   },
+  'booked-travel': {
+    isPageHeading: 'true',
+    mixin: 'radio-group',
+    className: ['govuk-radios'],
+    validate: ['required'],
+    options: [
+      {
+        value: 'yes',
+        toggle: 'booked-travel-date-to-uk-content',
+        child: 'partials/booked-travel-date-to-uk'
+      },
+      {
+        value: 'no'
+      }
+    ],
+    legend: {
+      className: 'govuk-!-margin-bottom-6'
+    }
+  },
+  'booked-travel-date-to-uk': dateComponent('booked-travel-date-to-uk', {
+    mixin: 'input-date',
+    validate: [
+      'required',
+      'date'
+    ],
+    validationLink: {
+      field: 'booked-travel',
+      value: 'yes'
+    }
+  }),
+  'travel-doc-number': {
+    className: ['govuk-input', 'govuk-!-width-one-third'],
+    validate: [
+      'required',
+      'alphanum',
+      { type: 'minlength', arguments: 9 },
+      { type: 'maxlength', arguments: 13 }
+    ]
+  },
+  'travel-doc-nationality': {
+    mixin: 'select',
+    className: ['typeahead'],
+    validate: ['required'],
+    options: [{
+      value: '',
+      label: 'fields.travel-doc-nationality.options.none_selected'
+    }].concat(countries)
+  },
+  'travel-doc-dob': dateComponent('travel-doc-dob', {
```

Comment:
Is it worth checking with BA/UCD for the two date component fields added to this PR because the user can enter impossible dates for these fields? I feel like they should be a bit more restrictive to stop users entering impossible dates? e.g.  01-01-1000 for these fields